dags/hivemind_etl_helpers/mediawiki_etl.py

```diff
@@ -0,0 +1,42 @@
+import logging
+
+from dags.hivemind_etl_helpers.ingestion_pipeline import CustomIngestionPipeline
+from dags.hivemind_etl_helpers.src.db.mediawiki.extractor import MediaWikiExtractor
+
+
+def process_mediawiki_etl(
+    community_id: str,
+    page_titles: list[str] | None = None,
+) -> None:
```

Comment:
The `api_url` will always be passed to this function.
Also, If there was no reason to make `page_titles` None, maybe we could just remove the equal to `None` of this variable?